### DISCUSSION TITLE
제페토 포스트 프로세싱 질문 입니다.

### DISCUSSION THREAD

**Discussion by mayalhc** [naverz/zepeto-studio-kor#444] [5. Q&A]
제페토 이번 업데이트로 포스트 프로세싱 적용이 가능한것으로 압니다
유니티 포스트 프로세싱 적용방법으로는 제페토 퍼팹 카메라 때문에 스크립트로 
적용했지만 오류가 발생합니다..
제페토 카메라를 커스텀으로 변경 가능하게 할수는 없을까요? 
제페토 플레이어 스크립트에 메인카메라 온/오프 기능을 추가하는 것도 좋을것 같습니다..

카메라 기능 처럼 예제를 제공해주시면 정말 감사 하겠습니다..

**Comment by mayalhc** [comment]
지금 빌드 등록후 아이폰 테스트 해봤습니다...
https://youtu.be/Q9mwkkdCjjo

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요~ 
제페토 카메라를 코드로 세팅하고 싶으신 거라면 다음과 같이 사용해 보세요 😄

- 변경할 카메라 = ZepetoPlayers.instance.LocalPlayer.zepetoPlayer.character;
